app.js

```diff
@@ -14,12 +15,29 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
-
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers', { beersData: beersFromApi });
+    })
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => {
+      console.log(responseFromAPI);
+      res.render('random-beer', { beersDataRandom: responseFromAPI });
```

Comment:
This is correct but you could also just pass `responseFromAPI[0] `to `res.render` since the `.getRandom() `api will always return an array of one element and so we can avoid the` #each` loop in the hbs.